datetimepickeredittext/src/main/java/io/blackbox_vision/datetimepickeredittext/internal/utils/DateUtils.kt

```diff
@@ -40,4 +40,12 @@ object DateUtils {
 
         return date
     }
+
+    private fun currentDT(): String {
+        return SimpleDateFormat(DATE_TEMPLATE).format(System.currentTimeMillis())
+    }
+
+    fun gettingDate(selDate: String): String {
```

Comment:
`selDate` per `strDate` would be more expressive.

Reply:
Definitely, that requires change. **strDate** would be more expressive

Reply:
Great, with this change, and the other in the first comment, I can merge it and release a new version.

Reply:
modified and pushed the code